spec/system/admin/account_requests_system_spec.rb

```diff
@@ -40,6 +40,19 @@
           expect(page).not_to have_content(request4.name)
         end
       end
+
+      it 'should close the account', js: true do
```

Comment:
Can we combine the expectations here with the request specs, so we don't need to do the same actions twice? We can just remove the request specs.

Reply:
I Couldn't test the validation errors in this spec because the button to "Close" is only displayed on the open requests, therefore I tested the validation on the controller spec, also I removed the duplicated tests in that controller to avoid the same action twice